src/test/scala/com/asidatascience/configuration/DynamicConfigurationFromHttpSpec.scala

```diff
@@ -49,14 +48,14 @@ with ScalaFutures {
 
   def withDynamicConfiguration(
     parser: TestConfigurationParser)(
-    block: DynamicConfiguration[Configuration] => Unit ): Unit = {
+    block: DynamicConfiguration[Configuration] => Assertion): Unit = {
```

Comment:
Should this be ` DynamicConfiguration[Configuration] => Any`, rather than to ` => Assertion`? Sometimes we like doing things that can cause the tests to throw, rather than explicitly returning an assertion (e.g. just calling `futureValue` on a future without checking the return type).